### PR TITLE
[14.0][MIG] sale_stock_return_request

### PR DESCRIPTION
**Pull request by dalonsod**
Standard migration from ~~v11~~ v15 + action access fix

### CODE REVIEW

**Comment by dalonsod** [comment]
Hello @chienandalu could you review?

And two things that I would ask you,

- I don't understand codecov issue, may you help me with it?
- I'll probably add a minor enhacement for this addon (add an optional filter by one ore more sales orders in return request), could I add it in this PR or better first merging this one?

Thanks in advance!

**Review by chienandalu — COMMENTED** [review]
Hi, @dalonsod . Thanks for your contributing :)

Out of curiosity, why are you migrating from 11.0? This module is already in 12, 13 and 15. You should be able to either migrate from 13 or 15 without issue as there aren't many changes involving this module across versions.

As for codecov, I wouldn't pay it too much attention :sweat_smile:

**Comment by dalonsod** [comment]
Thanks, I've simply looked for this addon, but in this case my search across repos was badly implemented :(

Please forget codecov issue, I'll make a new migration from v13 or v15 (the most updated one), so that problem may be gone

**Comment by rousseldenis** [comment]
@dalonsod ping

**Comment by dalonsod** [comment]
@sergio-teruel @cesar-tecnativa could you review?

I've tried to find out codecov problem, but I don't understand why coverage is incomplete (tests seem to cover everything)

File: sale_stock_return_request/models/stock_return_request.py [hunk 1]
```diff
@@ -27,15 +27,14 @@ def _action_confirm(self):
 
     def action_view_sales(self):
         """Display returned sales"""
-        action = self.env.ref("sale.action_orders")
-        result = action.read()[0]
-        result["context"] = {}
+        action = self.env["ir.actions.act_window"]._for_xml_id("sale.action_orders")
+        action["context"] = {}
```
Comment (chienandalu) [inline sale_stock_return_request/models/stock_return_request.py:31]: Squash these changes with your migration commit

**Review by chienandalu — APPROVED** [review]
Minor comment

File: sale_stock_return_request/models/stock_return_request.py (hunk 1 above)
Comment (dalonsod) [reply sale_stock_return_request/models/stock_return_request.py:31]: I've made that fix in a separate commit, so it could be cherry-picked to v15

Comment (dalonsod) [reply sale_stock_return_request/models/stock_return_request.py:31]: Indeed, made the same fix for purchase cousin: https://github.com/OCA/purchase-workflow/pull/2029

Comment (chienandalu) [reply sale_stock_return_request/models/stock_return_request.py:31]: Yes, but you can simply include it in your migration commit for this one

Comment (dalonsod) [reply sale_stock_return_request/models/stock_return_request.py:31]: Done!

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**